src/panels/lovelace/cards/hui-shopping-list-card.ts

```diff
@@ -132,6 +132,7 @@ class HuiShoppingListCard
               "ui.panel.lovelace.cards.shopping-list.add_item"
             )}
             @keydown=${this._addKeyPress}
+            .inputProps=${{ enterkeyhint: "done" }}
```

Comment:
This is from MUI I think? This is not how our components work,`enterkeyhint` would need to be implemented in `ha-textfield`, and then this should just be set as a property or attribute

Reply:
yes, im sorry, i seemed to have mixed up the frameworks when reading the code